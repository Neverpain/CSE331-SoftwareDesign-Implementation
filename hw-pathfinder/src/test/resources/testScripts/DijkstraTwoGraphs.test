# Tests that Dijkstra's algorithm works properly on two
# separate graphs

CreateGraph g1
CreateGraph g2

AddNode g1 A
AddNode g1 B
AddNode g2 A
AddNode g2 B

AddEdge g1 A B 1.0
AddEdge g2 A B 1.0

FindPath g1 A B

FindPath g2 A B
