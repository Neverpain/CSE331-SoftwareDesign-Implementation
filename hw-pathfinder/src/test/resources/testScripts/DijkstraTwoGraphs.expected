# Tests that Dijkstra's algorithm works properly on two
# separate graphs

created graph g1
created graph g2

added node A to g1
added node B to g1
added node A to g2
added node B to g2

added edge 1.000 from A to B in g1
added edge 1.000 from A to B in g2

path from A to B:
A to B with weight 1.000
total cost: 1.000

path from A to B:
A to B with weight 1.000
total cost: 1.000